validators/cgroup_validator_linux.go

```diff
@@ -68,6 +69,7 @@ func (c *CgroupsValidator) Validate(spec SysSpec) (warns, errs []error) {
 		requiredCgroupSpec = spec.CgroupsV2
 		optionalCgroupSpec = spec.CgroupsV2Optional
 	} else {
+		warns = append(warns, errors.New("cgroup v1 support is in maintenance mode"))
```

Comment:
cgroup -> cgroups

should we say something like:
```suggestion
		warns = append(warns, errors.New("cgroup v1 support is in maintenance mode. Please upgrade to cgroups v2"))
```
?

Reply:
Use https://github.com/kubernetes/kubernetes/pull/125328/files#diff-31b906a36201f455ea92866cf7adbd5440cb188fb06515b6e3d01772daec3f8dR47

> Cgroup v1 support is in maintenance mode, please migrate to Cgroup v2.

According to https://man7.org/linux/man-pages/man7/cgroups.7.html, we should use cgroups.

Reply:
yeah `cgroups` is the right name (plural noun with `s`)